### PR TITLE
Brigade 2022 Annual Review

### PR DESCRIPTION
**Pull request by krancour**
Signed-off-by: Kent Rancourt <kent.rancourt@microsoft.com>

### CODE REVIEW

**Comment by dims** [comment]
Folks, @mattfarina will be the TOC shepherd for the annual review meeting!

**Review by mattfarina — COMMENTED** [review]
In the TOC meeting today where we reviewed Brigade, we noted we are ok with Brigade being handed off to new maintainer. But, we will want to see that the project is taken over and active by KubeCon/CloudNativeCon NA in Detroit. We can review it at that time to see if it should be archived or continue.

See the meeting on 2022-06-21 at https://www.youtube.com/playlist?list=PLj6h78yzYM2Mf6GCZzW6CAk6GlZESbemB for more details.

**Comment by jorgearteiro** [comment]
As proposed new maintainer, I started onboarding on this project. I got access to some of the resources used by the previous maintainers, but I haven't had a proper handover. At this stage, I can't see this project going forward, as we need Go lang developers with some understanding of Brigade's components to keep updating and moving forward. Without deep support from previous maintainers for a period of time, a full time maintainer/project manager is required for the project. As I don't have this capacity available to contribute or to search for new contributors, I recommend to follow previous maintainers recommendation to Archive the project. 
Please nominate someone from CNCF to manage this transition period, and please remove any previous grant/access given to me for this project. Thanks in advance.

**Comment by krancour** [comment]
I'm not looking to pick a fight, especially in as public a forum as this. This is one of those moments where I know the professional thing to do is probably shut my mouth, but I'm also reluctant to stand by idly while myself or any other previous maintainer of the project is blamed for the current maintainer's failures.

> As proposed new maintainer

Let's be clear that you nominated yourself.

> At this stage, I can't see this project going forward, as we need Go lang developers with some understanding of Brigade's components to keep updating and moving forward.

Well... there's a reason that the usual order of things is that a contributor demonstrates the requisite deep technical knowledge of a project _before_ becoming a maintainer. You had every opportunity to acquire this knowledge _prior_ to becoming a maintainer by making more extensive technical contributions to the project, but you did not avail yourself of those opportunities.

The bottom line here is that I only want to set the record straight. The previous maintainers, myself included, didn't put you in this position by failing to support you through a transition. _You_ put yourself in this position by insisting that the TOC hand you responsibility you were not prepared for, even after the previous maintainers recused themselves from doing just that.

**Comment by jorgearteiro** [comment]
Just to be clear, previous maintainers, mostly full time and paid, did a great job on Brigade v1 and v2. This is the reason i believe on this project in a first place. I used Brigade since early days. I put my hands-up on Slack channel exactly for this reason. I tried to get support in my free time, but was not successful to justify keeping the project. Thanks @krancour for publishing Brigade v2 pretty much by himself, please don't blame me for trying mate! I hope you still remember all hours i put to translate documentation, create cron event source and promoted the project multiple times(events, videos, conferences, office hours(6am on my time zone, etc).

**Comment by jorgearteiro** [comment]
> The bottom line here is that I only want to set the record straight. The previous maintainers, myself included, didn't put you in this position by failing to support you through a transition. _You_ put yourself in this position by insisting that the TOC hand you responsibility you were not prepared for, even after the previous maintainers recused themselves from doing just that.

Also to be clear, i never talked with TOC, and i can't see my name been nominated on this PR or on TOC's meeting recording. I haven't insisted at all. I guess i was their last resort to keep the project alive. I am more than prepared to take over this project, i have done a few community meetings and conference(including weekends) to invite new contributors, as you can see on the last meeting records(https://hackmd.io/KyyZMHyzSzGEiQY0ZHPqgw?view), and i am very grateful for them for giving a shot as well. We are all very capable to maintain this project, but we have to decide not to do, and we will work in other OSS projects instead where we can have support.
